docs/contributing/code_of_conduct.rst

```diff
@@ -13,7 +13,7 @@ We will not tolerate abusive behaviour or language or any form of harassment.
 
 Individuals whose behaviour is a cause for concern will be give a warning, and
 if necessary will be excluded from participation in official django CMS
-channels (email lists, IRC channels, etc) and events. The `Django Software
+channels (email lists, Slack/IRC channels, etc) and events. The `Django Software
```

Comment:
channels (forum, Slack, etc)

Reply:
Added Discourse, but retained the mention of email lists and IRC, since they still exist.